tests/functional/tests/bundles/overlays/jammy.yaml.j2

```diff
@@ -0,0 +1,6 @@
+series: jammy
+applications:
+  prometheus:
+    series: focal
```

Comment:
Is it right that prometheus should be on focal in a jammy deployment?

Reply:
I don't know, but that's not something I changed, so it's really out of scope for this PR - see [the file on main for reference](https://github.com/canonical/charm-prometheus-blackbox-exporter/blob/7bff4ec5e49bb9dc11c279817e068c0758dd3c53/src/tests/functional/tests/bundles/overlays/jammy.yaml.j2#L1). :)

Reply:
Behalf of @gabrielcocenza to resolve the conversation since it's out of scope in this PR.